scripts/superdesk/editor2/add-embed.ctrl.js

```diff
@@ -71,9 +71,13 @@ EMBED_PROVIDERS, $scope, editor, config, $injector) {
                         name: EMBED_PROVIDERS.youtube
                     }
                 ];
-                // prepend with custom handlers from config
-                if (config.editorEmbedCodeParsers) {
-                    knownProviders = $injector.invoke(config.editorEmbedCodeParsers).concat(knownProviders);
```

Comment:
I need then to move https://github.com/superdesk/superdesk/blob/pa-poc/server/pa/vidible.py to `superdesk-core`